src/app/(auth)/(navigationsBarLayout)/mypage/ui/ModifyForm.tsx

```
@@ -0,0 +1,164 @@
+              className="cursor-pointer"
+            />
+
+            <input
+              type="file"
+              ref={fileInputRef}
+              className="hidden"
+              accept="image/*"
+              onChange={handleFileChange}
+            />
+            <h4 className="text-lg font-bold text-white text-center"></h4>
+            <div className=" text-white text-2xl  text-center">{myInfo.nickname}</div>
```

Biome (2.1.2)

[error] Provide screen reader accessible content when using heading  elements.

All headings on a page should have content that is accessible to screen readers.

(lint/a11y/useHeadingContent)